source/chapters/library.rst

```diff
@@ -324,6 +326,9 @@ Tracks can be loaded into a deck in several ways:
   example, on macOS, dragging-and-dropping a track from iTunes onto one of
   Mixxx's waveform displays will load it into that deck.
 
+
+Unloading Tracks
+
```

Comment:
Your previous additions are missing
```suggestion

Unloading Tracks
* To unload tracks use the keyboard shortcuts :kbd:`Shift` + :kbd:`Alt` + :kbd:`Left` for Deck 1 and :kbd:`Shift` + :kbd:`Alt` + :kbd:`Right` for Deck 2.

* You can also use the eject track button to unload track from a deck. Look for a button with this icon:

  .. figure:: ../_static/Mixxx-eject-button.png
     :align: center
     :width: 100%
     :figwidth: 100%
     :alt: Mixxx library - Eject Button
     :figclass: pretty-figures
```

Reply:
You asked me to remove the picture

Reply:
sorry, I'm confused some reviews.
I trimmed the suggestion above. please check.

We can add the picture later on, so this get's merged soonish.

Also I went through previous reviews so we don't miss anything and don't have contradicting change proposal.
Sorry for the hazzle!